com.microsoft.mrtk.graphicstools.unity/Samples~/UniversalRenderPipelineAsset_Renderer.asset

```diff
@@ -1,54 +0,0 @@
-%YAML 1.1
```

Comment:
Did you mean to delete this? Might be fixed when you reconcile "Samples" and "Samples~"